File: Assets/Scripts/GameToManage.cs

// Hannes Helmholz
//
// abstract class for all games managed by GameController

using UnityEngine;

[RequireComponent (typeof (NetworkView))] // for RPC

public /*abstract*/ class GameToManage : MonoBehaviour {
	
	public void Play() {}
	public void Stop() {}
	public void Reset() {}
}

File: ISS Emergency/Assets/Scripts/GameToManage.js
#pragma strict

// Hannes Helmholz
//
// abstract class for all games managed by GameController


@script RequireComponent(NetworkView) // for RPC

private /*abstract*/ class GameToManage extends MonoBehaviour {
	
	public function Play() {}
	public function Stop() {}
	public function Reset() {}
}